projects/igniteui-angular/src/lib/select/select.component.ts

```diff
@@ -269,7 +269,6 @@ export class IgxSelectComponent extends IgxDropDownComponent implements IgxSelec
     ngAfterContentInit() {
         this._overlayDefaults = {
             modal: false,
-            closeOnOutsideClick: true,
```

Comment:
This should be explicit false though, as the default settings of the underlying overlay service have that enabled.